Dockerfile

```diff
@@ -30,6 +30,9 @@ ARG TOMCAT_JRE=jdk8
 # Use official maven image for the build
 FROM maven:3-jdk-8 AS builder
 
+# Install chromium-driver for sake of JavaScript unit tests
+RUN apt update && apt install -y chromium-driver
```

Comment:
We're already apt-updating on line 57 below - do we need to do it twice?

Reply:
Also, should this be `apt update` and then `apt install`, or `apt-get update` and `apt-get install`?

Reply:
Good point @necouchman - should probably be `apt-get`. `apt` is specifically discouraged from being used in scripts by the developers.

Reply:
> We're already apt-updating on line 57 below - do we need to do it twice?

Yes - this is for the build image of a multi-stage build, whereas line 57 is the runtime image.

Reply:
OK - amended the commit to use `apt-get`.